README.md

```diff
@@ -427,6 +421,42 @@ NOTE
 - `DeepOmit` works fine with `Array`s and `Set`s. When applied to a `Map`, the filter is only applied to its value.
 - If there exists any property in the filter which is not in the original type, an error will occur.
 
+### DeepPick
+
+Recursively pick deep properties according to key names.
+
+This type works as complementary type to DeepOmit, in the similar way like Exclude and Extract types complement each other.
+
+The filter syntax is the same as for the DeepPick, so one filter can be used to obtain both DeepPick and DeepOmit types from it.
+
+The properties to be picked completely should be defined as `never`. For the properties you want to partially pick, you should recursively define the sub-properties to be picked.
+
+```typescript
+interface Teacher {
+  name: string;
+  gender: string;
+  students: { name: string; score: number }[];
+}
+```
+
+```typescript
+type TeacherSimple = DeepPick<
+  Teacher,
+  {
+    gender: never;
```

Comment:
Now we have `true` filter instead of `never` AFAIK

```suggestion
    gender: true;
```

Don't forget to change it below too